scripts/api_diff_report/api_diff_report.py

```diff
@@ -0,0 +1,259 @@
+# Copyright 2023 Google LLC
```

Comment:
> theme dir: Jazzy documentation theme. in case that Jazzy changes it's default theme.

This bit makes it sound like the version of jazzy used may change. Is this likely? If not, what do you think of just installing a specific jazzy version on ci?

Reply:
Will install jazzy [v0.14.3](https://github.com/realm/jazzy/releases/tag/v0.14.3) in the workflow